terragrunt/modules/crates-io/compute-static/src/main.rs

```diff
@@ -127,6 +130,28 @@ fn rewrite_urls_with_plus_character(request: &mut Request) {
     }
 }
 
+/// Rewrite `/crates/{crate}/{version}/download` URLs to
+/// `/crates/{crate}/{crate}-{version}.crate`
+///
+/// cargo versions before 1.24 don't support placeholders in the `dl` field
+/// of the index, so we need to rewrite the download URL to point to the
+/// crate file instead.
+fn rewrite_download_urls(request: &mut Request) {
+    static RE: Lazy<Regex> = Lazy::new(|| {
+        Regex::new(r"^/crates/(?P<crate>[^/]+)/(?P<version>[^/]+)/download$").unwrap()
+    });
```

Comment:
I'd prefer moving this into a constant, but also seem to vaguely remember that `once_cell` doesn't support that (yet)?

Reply:
I don't think const is possible since the regex can't currently be constructed at buildtime

Reply:
Yeah that's what I thought. Then let's leave it like this for now. 👍